tests/console-backend-service/internal/domain/servicecatalog/servicecatalog_test.go

```diff
@@ -24,18 +25,22 @@ func TestMain(m *testing.M) {
 
 	module.SkipPluggableMainIfShould(c, ModuleName)
 
-	scInstaller, err := setup.NewServiceCatalogInstaller("console-backend-service-sc")
-	exitOnError(err, "while initializing Service Catalog installer")
+	scInstaller, err := setup.NewServiceCatalogConfigurer(TestNamespace, true)
+	exitOnError(err, fmt.Sprintf("while initializing Service Catalog Configurer for module %s", ModuleName))
```

Comment:
Just add formatting to `exitOnError`. It will be easier to use :smile:

Please consider it also in other places.